pkgs/development/python-modules/posting/default.nix

```
@@ -0,0 +1,112 @@
+{
+  buildPythonPackage,
+  hatchling,
+  pkgs,
+  fetchFromGitHub,
+  python3Packages,
```

```suggestion
  httpx,
      textual-autocomplete,
      openapi-pydantic,
      click,
      xdg-base-dirs,
      click-default-group,
      pyperclip,
      pyyaml,
      pydantic-settings,
      python-dotenv,
      watchfiles,
      rich,
      toml,
      pydantic,
      typer,
```
Using python3Packages in python-modules is not allowed.